code/game/machinery/computer/camera.dm

```diff
@@ -60,9 +62,13 @@
 /obj/machinery/computer/security/ui_interact(mob/user, datum/tgui/ui)
 	// Update UI
 	ui = SStgui.try_update_ui(user, src, ui)
-	// Show static if can't use the camera
-	if(!active_camera?.can_use())
+
+	// Update the camera, showing static if we can't use it and updating data if the location has moved.
+	if(active_camera)
+		update_active_camera_screen()
+	else
 		show_camera_static()
```

Comment:
I don't like that you replaced `if(!active_camera?.can_use())` with two if statements (here and one in the proc). It only adds more loc and complexity. Just revert to how it was before (with .?)

Reply:
Change made.